### PR TITLE
Added Cody AI Assistant to Code Editing section

### PR DESCRIPTION
**Pull request by Prajwal-Shah**
## Requirements

 * [x] This is Software as a Service not self hosted 
         It a Free SaaS Product.
 * [x] It has a free tier not just a free trial
         It has a free tier.
 * [x] Pricing information is clearly visible without signup or phone calls
         Free tier pricing listed on website - https://sourcegraph.com/pricing
 * [x] The submission mentions what is free.
         Listed on pricing page- https://sourcegraph.com/pricing
 * [x] The submission is not already present in the list
         Not present in list.
 * [x] The service has contact details of those running it and a privacy policy
         Contact page - https://sourcegraph.com/contact
         Privacy policy - https://sourcegraph.com/terms/privacy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1605,6 +1605,10 @@ Update Time, five active automations, webhooks.
    * [Brackets](http://brackets.io/) - Brackets is an open-source text editor specifically designed for web development. It is lightweight, easy to use, and highly customizable.
    * [cacher.io](https://www.cacher.io) — Code snippet organizer with labels and support for 100+ programming languages.
    * [Code::Blocks](https://codeblocks.org) — Free Fortran & C/C++ IDE. Open Source and runs on Windows,macOS & Linux.
+   * [Cody](https://sourcegraph.com/cody) - Free AI coding assistant that can write (Code blocks, autocomplete, unit tests), understand (knowledge of your entire codebase), fix, and find your code.
```
Comment (ripienaar) [inline README.md:1608]: As you can see entries are 1 line only except for major cloud providers etc.  Please follow the same style

Comment (Prajwal-Shah) [reply README.md]: Sure, fixing it.

Comment (Prajwal-Shah) [reply README.md]: Made the changes into oneline

**Review by Prajwal-Shah — COMMENTED** [review]
Made the changes as suggested.

**Comment by ripienaar** [comment]
Thanks

**Pull request merged**